src/gui/touchscreengui.cpp

```diff
@@ -36,23 +36,144 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <iostream>
 #include <algorithm>
 
-using namespace irr::core;
-
 TouchScreenGUI *g_touchscreengui;
 
 const std::string button_image_names[] = {
```

Comment:
can be `static const char *button_image_names[]` too btw

Reply:
it was until #13640 🤷, will change it back